backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/fab/BpkFab.kt

```diff
@@ -44,14 +44,15 @@ fun BpkFab(
         interactionSource = interactionSource,
         modifier = modifier,
         shape = CircleShape,
-        backgroundColor = BpkTheme.colors.coreAccent,
+        containerColor = BpkTheme.colors.coreAccent,
         contentColor = BpkTheme.colors.textPrimaryInverse,
         elevation = FloatingActionButtonDefaults.elevation(BpkElevation.Xl, BpkElevation.Xl, BpkElevation.Xl, BpkElevation.Xl),
     ) {
         BpkIcon(
             icon = icon,
             contentDescription = contentDescription,
             size = BpkIconSize.Large,
+            tint = androidx.compose.material3.LocalContentColor.current,
```

Comment:
Another case of containered widget which sets MD3 CompositionLocal – but BpkIcon takes MD2 local for now.